astropy/modeling/tests/test_fitters.py

```diff
@@ -291,6 +326,35 @@ def test_linear_fit_model_set_fixed_parameter(self):
         assert_allclose(fitted_model.c1, [1., -2.], atol=1e-14)
         assert_allclose(fitted_model.c2, [0.5, 0.], atol=1e-14)
 
+    def test_linear_fit_2d_model_set_common_weight(self):
+        init_model = models.Polynomial2D(degree=2, c1_0=[1, 2], c0_1=[-0.5, 1],
+                                         n_models=2,
+                                         fixed={'c1_0': True, 'c0_1': True})
```

Comment:
Just wondering whether there's any special reason for fixing the parameters here and in the other test?